pkg/packaging/packaging.go

```diff
@@ -754,6 +771,21 @@ sleep 5
 	return t.ExecuteTemplate(w, "postinstall", options)
 }
 
+type newSyslogTemplateOptions struct {
+	LogPath string
+	PidPath string
+}
+
+func renderNewSyslogConfig(w io.Writer, options *newSyslogTemplateOptions) error {
+	syslogTemplate := `# logfilename          [owner:group]    mode count size when  flags [/pid_file] [sig_num]
+{{.LogPath}}               640  4000     3   *   G  {{.PidPath}} 15`
```

Comment:
This says keep 4,000 3k files? That seems like a weird choice.

Reply:
🤦‍♂️ 

I meant 3 4000 kb files